editor/scene_tree_dock.cpp

```diff
@@ -4093,10 +4271,24 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	delete_tracks_checkbox->set_pressed(true);
 	vb->add_child(delete_tracks_checkbox);
 
+	editable_instance_enable_or_disable_dialog = memnew(ConfirmationDialog);
+	editable_instance_enable_or_disable_dialog->set_ok_button_text("Enable for all");
+	editable_instance_disable_all_button = editable_instance_enable_or_disable_dialog->add_button("Disable for all", true);
+	add_child(editable_instance_enable_or_disable_dialog);
+	editable_instance_disable_all_button->connect("pressed", callable_mp(this, &SceneTreeDock::_disable_all_editable_children_from_selection));
+	editable_instance_enable_or_disable_dialog->connect("confirmed", callable_mp(this, &SceneTreeDock::_enable_all_editable_children_from_selection));
+
 	editable_instance_remove_dialog = memnew(ConfirmationDialog);
 	add_child(editable_instance_remove_dialog);
 	editable_instance_remove_dialog->connect("confirmed", callable_mp(this, &SceneTreeDock::_toggle_editable_children_from_selection));
 
+	placeholder_enable_or_disable_dialog = memnew(ConfirmationDialog);
+	placeholder_enable_or_disable_dialog->set_ok_button_text("Enable for all");
```

Comment:
```suggestion
	placeholder_enable_or_disable_dialog->set_ok_button_text(TTR("Enable for all"));
```
Same for all buttons.